src/manifold/src/properties.cpp

```diff
@@ -387,9 +382,9 @@ void Manifold::Impl::CalculateBBox() {
  */
 bool Manifold::Impl::IsFinite() const {
   auto policy = autoPolicy(NumVert());
-  return transform_reduce<bool>(policy, vertPos_.begin(), vertPos_.end(),
-                                FiniteVert(), true,
-                                thrust::logical_and<bool>());
+  return reduce<bool>(policy, TransformIterator(vertPos_.begin(), FiniteVert()),
```

Comment:
Why do you prefer `reduce` with a transform iterator to `transform_reduce`?

Reply:
Oh, I forgot there is such a thing. Will fix.

Reply:
I think I have this change in a commit of #823 if you want to grab it from there.